public/index.php

```diff
@@ -0,0 +1,56 @@
+<?php
+if (!isset($_POST["button"])) {
+    $url = $_POST["url"];
+    $delay = $_POST["delay"];
+    $size = $_POST["size"];
+
+    $timestamp = time();
+
+    function MakeDirectory($timestamp)
+    {
+        if (!mkdir("image/" . $timestamp, 0777, true)) {
+            return false;
+        }
+    }
+
+
+    function GenerateImage($timestamp, $url, $delay, $size)
+    {
+        exec("cd image/" . $timestamp . " && " . "/usr/local/bin/node ../../../index.js"
+            . " " . $url
```

Comment:
If there's a space in the url (eg. a literal space or a %20, depending on whether it was escaped correctly) then what happens here? Could this be exploited?